File: 프로그래머스/0/120845. 주사위의 개수/주사위의 개수.java
class Solution {
    public int solution(int[] box, int n) {
        int answer = 1;
        
        for(int side : box) {
            answer *= side / n;
        }
        
        return answer;
    }
}

// 다른 풀이
int answer = 1;

answer *= box[0]/n;
answer *= box[1]/n;
answer *= box[2]/n;

return answer;

File: 프로그래머스/0/120845. 주사위의 개수/주사위의 개수.py
def solution(box, n):
    answer = 1
    
    for side in box:
        answer *= side // n
    
    return answer

# 다른 풀이
x, y, z = box
return (x // n) * (y // n) * (z // n )

# 다른 풀이
return math.prod(map(lambda v: v//n, box)) # import math

# 다른 풀이
return (box[0] // n) * (box[1] // n) * (box[2] // n)
